tests/tracer/test_helpers.py

```diff
@@ -1,54 +0,0 @@
-import mock
-
-from ddtrace import helpers
```

Comment:
as long as `helpers` exists, we should keep these tests

Reply:
I renamed `test_helpers.py` into `test_correlation_ids.py` since `helpers` is deprecated and the same functionality was moved over to Tracer 😕 I'm open to keeping the tests here, but thought it was more relevant to change the tests to be relevant to tracer tests...

Reply:
I think the concern I have, is people are going to keep using the `helpers.get_correlation_ids()` for awhile until they either notice the deprecation or have a chance to update their usage.

in which case, we should ensure the functionality continues to work while we still support it until 1.0.0

Reply:
So duplicate tests is fine until 1.0.0?

Reply:
yup! I think so.

if we wanted to be 100% sure the functionality is the same we could always use `helpers.get_correlation_ids()` in the tracer tests to assert the results are the same?

```python
trace_id, span_id = self.tracer.get_correlation_ids()
assert trace_id == <expected>
assert span_id == <expected>

# Remove me when we remove helpers module
assert (trace_id, span_id) == helpers.get_correlation_ids(self.tracer)
```

maybe?